app/controllers/vm_common.rb

```diff
@@ -115,14 +115,22 @@ def launch_vmware_console
                   :mks_classid => get_vmdb_config[:server][:mks_classid]
                 )
               when "vmrc"
+                host = @record.ext_management_system.ipaddress || @record.ext_management_system.hostname
+                ticket = j(params[:ticket])
```

Comment:
Should this be wrapped in `j` if it's being build in the Ruby side?  Or perhaps stated differently, should the vmrc_uri be wrapped in `j` in its entirey?